<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>maskSettingsDialog</class>
 <widget class="QWidget" name="maskSettingsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>592</width>
    <height>472</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="0" colspan="3">
    <layout class="QHBoxLayout" name="typeChangeLayout">
     <property name="spacing">
      <number>26</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetMaximumSize</enum>
     </property>
     <item>
      <widget class="QCheckBox" name="logCheckBox">
       <property name="text">
        <string>View change to main log</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="drawGraphCheckBox">
       <property name="text">
        <string>Draw a graph</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QGroupBox" name="numberSettingsBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>NumBox</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="decimalIntLabel">
        <property name="text">
         <string>Decimal integer</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="shiftlabel">
        <property name="text">
         <string>Shift</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="shiftTxt"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="koeffLabel">
        <property name="text">
         <string>*K</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="koeffTxt"/>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="decimalInt">
        <property name="text">
         <string>&lt;waiting new data...&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="uncheckAllButton">
     <property name="text">
      <string>Uncheck all</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="checkAllButton">
     <property name="text">
      <string>Check all</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QGroupBox" name="dynamicMaskForm">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>BitBox</string>
     </property>
     <layout class="QGridLayout" name="dynamicMaskFormLay">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="wordInfo">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>wordInfoLabel</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QLineEdit" name="maskName"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
